ballerina/llm_client_default.bal

```diff
@@ -47,7 +47,17 @@ public isolated distinct client class DefaultBallerinaAzureOpenAIModel {
         };
 
         http:Client cl = self.cl;
-        chat:CreateChatCompletionResponse chatResult = check cl->/chat/complete.post(chatBody);
+        http:Response chatResponse = check cl->/chat/complete.post(chatBody);
+        if (chatResponse.statusCode == 401 || chatResponse.statusCode == 403) {
```

Comment:
Shouldn't this be only 403?

Reply:
401 means unauthorized, So user needs to change the token

Reply:
401 Unauthorized
Meaning: The request lacks valid authentication credentials, or the provided credentials are invalid or expired.
403 Forbidden
Meaning: The server understands the request, but the client does not have permission to access the requested resource.

I will update only for 401